extensions/blocks/podcast-player/podcast-player.php

```diff
@@ -98,16 +100,26 @@ function render_player( $track_list, $attributes ) {
 	}
 
 	ob_start();
-
+	$initial_track_src = ! empty( $track_list[0]['src'] ) ? $track_list[0]['src'] : '';
 	?>
-	<div class="<?php echo esc_attr( $block_classname ); ?>">
-		<?php // Placeholder: block markup is being handled in https://github.com/Automattic/jetpack/pull/14952. ?>
-		<script type="application/json" class="wp-playlist-script"><?php echo wp_json_encode( $player_data ); ?></script>
+	<div class="<?php echo esc_attr( $block_classname ); ?>" id="<?php echo esc_attr( $instance_id ); ?>">
+		<audio src="<?php echo esc_attr( $initial_track_src ); ?>" preload="none"></audio>
+		<ol>
+			<?php
+			foreach ( $track_list as $att_id => $attachment ) :
+				printf( '<li><a href="%1$s" data-jetpack-podcast-audio="%2$s">%3$s</a></li>', esc_url( $attachment['link'] ), esc_url( $attachment['src'] ), esc_html( $attachment['title'] ) );
+			endforeach;
+			?>
+		</ol>
+		<script type="application/json"><?php echo wp_json_encode( $player_data ); ?></script>
 	</div>
+	<script>window.jetpackPodcastPlayers=(window.jetpackPodcastPlayers||[]);window.jetpackPodcastPlayers.push( <?php echo wp_json_encode( $instance_id ); ?> );</script>
 	<?php
 	/*
 	* Enqueue necessary scripts and styles.
 	*/
+	wp_enqueue_script( 'wp-mediaelement' );
+	wp_enqueue_style( 'wp-mediaelement' );
```

Comment:
Is this direct enqueue ok like this? `load_assets_as_required` doesn't offer options we would need to add these as our dependencies.

Reply:
It doesn't seem ideal since we can't link these as a dependency for our added scripts. When this PR that fixes `load_assets_as_required` gets merged, we could use it: https://github.com/Automattic/jetpack/pull/14958

Either way, `wp-mediaelement` includes extra styles and JS we don't need from core. It'd be nice to find a way to not bring those unnecessary things in.

Reply:
Found the reference to just the mediaelement script without WP core's implementation: https://core.trac.wordpress.org/browser/tags/5.3/src/wp-includes/script-loader.php#L1298